Assets/Scripts/StaminaBar.cs

```diff
@@ -24,20 +27,13 @@ public class StaminaBar : MonoBehaviour {
 	}
 
     // Move is called each step of the players movement
-    // Returns state of stamina bar (crash or not);
+    // Returns if player can move;
     public bool move()

```

Comment:
The card simply says 'Pickup - Stamina Boost', also the pull is named `Add: Pickup Stamina Booster` and the branch is named 'stamina-pickup'. So you're saying It **shouldn't** pick up?